ert_shared/ensemble_evaluator/prefect_ensemble/prefect_ensemble.py

```diff
@@ -270,30 +270,41 @@ def _fetch_input_files(self):
         return input_files
 
     def _evaluate(self, dispatch_url, ee_id):
-        sys.stderr = open("prefect.log", "w")
+        try:
```

Comment:
A couple of questions:

1. Isn't this catchall a bit too coarse?
1. If `send` fails inside the `try` block, what happens when `send` fails in the `except` block?
1. If these exceptions are caught here, is not the `EnsembleEvaluator` (the caller) left a bit in the dark as to what happened to the evaluation?
1. All ensembles (legacy, prefect) should behave in the same-ish way. You're not adding this to the legacy ensemble, does that mean it's already there, or that it's coming?

Reply:
Good questions, remember its a WIP, some of the questions are still open:

1. For sure, generally that is not good. But the point is that exceptions that occur here will be lost because this function runs in a separate process. When not caught, the only thing you could get is some exit code. So, we want to catch exceptions, serialize them and send them to the parent to handle.
2. Yes, I have not figured out what to do about that.Basically if that happens you only have the exit status of the process left to look at. Which in this code is not easy, since we are not waiting for the sub-process to finish (no join), we happily soldier on and listen to events. One could at least try to retry a few time I guess, but that is not great either....
3. Yes, this code has not added yet any serialization of the caught exception, nor anything to handle that by the parent...
4. Yes, when we figure out how to handle the errors, it should be the same anywhere. However, I cannot move the handling out of the caller by not catching errors here, then you lose them. So that is why they are caught and translated to an event. Then, they should be handled in some uniform way.

It's a WIP, I have not figured out how to do this best, so any suggestions are very welcome...